translations/NDRC Welsh Translation_20210810.csv

```diff
@@ -1,728 +1,1521 @@
+"claimantType.02"	"No, I am a representative of the importer"	"Na, fi yw cynrychiolydd y mewnforiwr"	Added.
+"claimantType.01"	"Yes"	"Iawn"	Added.
+"reasonForOverpayment.hint"	"Provide as much information as possible. This will help us to process your application faster."	"Darparwch gymaint o wybodaeth â phosibl. Bydd hyn yn ein helpu i brosesu’ch cais yn gyflymach."	Added.
+"amend.application.closed.p2"	"If you feel we closed this application in error you can email us at {0} with the additional information."	"Os ydych yn teimlo ein bod wedi cau’r cais drwy gamgymeriad, gallwch anfon e-bost atom yn {0} gyda’r wybodaeth ychwanegol."	Added.
+"impInfo.checkYourAnswersLabel"	"Important information"	"Gwybodaeth bwysig"	Added.
+"numberOfEntriesType.02.hint"	"Multiple entries must all be from the same importer, involve the same reasons for the application and be from before 1 January 2021 or from that date onwards."	"Mae’n rhaid i gofnodion lluosog ddod o’r un mewnforiwr, cynnwys yr un rhesymau dros y cais a bod o gyn 1 Ionawr 2021 neu o’r dyddiad hwnnw ymlaen."	Added.
+"claimReason.documents.proof_of_return"	"proof of return"	"tystiolaeth o’r adenillion"	Added.
+"claimReason.documents.proof_of_quota"	"proof of the quota and commodity codes that are being claimed against"	"tystiolaeth o’r cwota a’r codau nwyddau y gwneir hawliad yn eu herbyn"	Added.
+"evidenceSupportingDocs.title"	"Documents we need from you"	"Dogfennau sydd eu hangen arnom gennych"	Added.
+"evidenceSupportingDocs.heading"	"Documents we need from you"	"Dogfennau sydd eu hangen arnom gennych"	Added.
+"claimantType.error.required"	"Confirm if you are the importer or their representative"	"Cadarnhewch ai’r mewnforiwr neu gynrychiolydd y mewnforiwr ydych chi"	Added.
+"claimReasonType.error.required"	"Select the main reason for making an application"	"Dewiswch y prif reswm dros wneud cais"	Added.
+"whomToPay.error.required"	"Select who you want HMRC to repay"	"Dewiswch pwy yr hoffech i CThEM ad-dalu"	Added.
+"furtherInformation.hintText"	"This may be information that HMRC has asked for, information you choose to add or a request to cancel your application."	"Gall hyn fod yn wybodaeth y mae CThEM wedi gofyn amdani, gwybodaeth y byddwch yn dewis ei hychwanegu neu gais i ganslo’ch cais."	Added.
+"entryDetails.importer.hint"	"You can find these on the E2 or C88 documents. If you’re unsure, you can speak to your shipping agent or courier."	"Gallwch ddod o hyd i’r rhain ar y dogfennau E2 neu C88. Os ydych yn ansicr, gallwch siarad â’ch asiant cludo neu’ch cludwr."	Added.
+"entryDetails.representative.hint"	"You can find these on the E2 or C88 documents."	"Gallwch ddod o hyd i’r rhain ar y dogfennau E2 neu C88."	Added.
+"enterAgentEORI.hint"	"This has 2 letters, followed by either 12 or 15 numbers. For example, GB123456789123 or GB123456789123000."	"Mae hwn yn cynnwys 2 lythyren, wedi’u dilyn gan naill ai 12 neu 15 rhif. Er enghraifft, ‘GB123456789123’ neu ‘GB123456789123000’."	Added.
```

Comment:
AMBER: Seems like the Welsh translation team have added quotes around these example, where the English does not have quotes around them (on purpose).

Reply:
Removed the single quotes from the "source" csv file and re-ran the migration tools